infrastructure/state.tf

```
@@ -0,0 +1,17 @@
+terraform {
+  required_version = ">= 0.14"
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 3"
+    }
+  }
+  backend "s3" {
+    bucket         = "etrusted-mars-terraform-state"
+    key            = "services/etrusted-api-tracking-datalake-sender/terraform.tfstate"
```

Key for the state should be equal to the repository name

```suggestion
    key            = "services/api-tracking-datalake-sender/terraform.tfstate"
```